src/app/app.component.html

```diff
@@ -1,11 +1,14 @@
 <h1>Components App</h1>
 <app-pizza-toppings></app-pizza-toppings>
+<app-ts-addition></app-ts-addition>
```

Comment:
Your component should be moved to the end...

And you should not change the order of the other components ! ! !